lumigator/python/mzai/backend/backend/tests/api/routes/test_datasets.py

```diff
@@ -113,3 +113,35 @@ def test_experiment_format_validation(
             files={"dataset": ("dataset.csv", d)},
         )
         assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+def test_upload_csv_delete(app_client: TestClient):
+    response = app_client.get("/health")
+    assert response.status_code == status.HTTP_200_OK
+    cwd = Path(__file__).parent
```

Comment:
Let's pull in all datasets as fixtures from a `conftest` file following the existing pattern here https://github.com/mozilla-ai/lumigator/blob/86cc96c2c8da01e8b1b798adc4f451f4a02149a6/lumigator/python/mzai/evaluator/evaluator/tests/conftest.py#L27